sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/ReceiptExtensions.java

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.formrecognizer;
+
+import com.azure.ai.formrecognizer.models.FormField;
+import com.azure.ai.formrecognizer.models.RecognizedReceipt;
+import com.azure.ai.formrecognizer.models.USReceipt;
+import com.azure.ai.formrecognizer.models.USReceiptItem;
+import com.azure.ai.formrecognizer.models.USReceiptType;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import static com.azure.ai.formrecognizer.models.ReceiptItemType.NAME;
+import static com.azure.ai.formrecognizer.models.ReceiptItemType.PRICE;
+import static com.azure.ai.formrecognizer.models.ReceiptItemType.QUANTITY;
+import static com.azure.ai.formrecognizer.models.ReceiptItemType.TOTAL_PRICE;
+
+/**
+ * The ReceiptExtensions class to allow users to convert a {@link RecognizedReceipt}
+ * to a receiptLocale specific Receipt type.
+ */
+@SuppressWarnings("unchecked")
```

Comment:
Prefer to have this suppresswarnings at the method level instead of having it for the entire class.